easybuild/easyconfigs/t/tensorstore/tensorstore-0.1.72-gfbf-2024a.eb

```diff
@@ -0,0 +1,397 @@
+# Thomas Hoffmann, EMBL Heidlelberg, structures-it@embl.de, 2025/03
+easyblock = 'PythonBundle'
+
+name = 'tensorstore'
+version = '0.1.72'
+
+homepage = 'https://github.com/google/tensorstore'
+description = """TensorStore is an open-source C++ and Python software library designed for
+storage and manipulation of large multi-dimensional arrays."""
+
+toolchain = {'name': 'gfbf', 'version': '2024a'}
+
+builddependencies = [
+    ('pybind11', '2.12.0'),
+    ('Bazel', '7.4.1', '-Java-21'),  # .bazelversion has 7.4.0
+    ('PyYAML', '6.0.2'),
+    ('NASM', '2.16.03'),
+    ('nlohmann_json', '3.11.3'),
+    ('snappy', '1.1.10'),
+]
+
+dependencies = [
+    ('Python', '3.12.3'),
+    ('SciPy-bundle', '2024.05'),
+    ('ml_dtypes', '0.5.0'),
+    ('zlib', '1.3.1'),
+    ('LibTIFF', '4.6.0'),
+    ('libpng', '1.6.43'),
+    ('libavif', '1.2.0'),
+    ('cURL', '8.7.1'),
+    ('bzip2', '1.0.8'),
+    ('BLAKE3-C', '1.6.1'),
+    ('libwebp', '1.4.0'),
+    ('Blosc', '1.21.6'),
+    ('tinyxml2', '10.0.0'),
+    ('c-ares', '1.34.4'),
+]
+
+_systemlibs = [
+    'numpy',
+    'com_github_pybind_pybind11',
+    'net_zlib',
+    'jpeg',
+    'png',
+    'org_lz4',
+    'libtiff',
+    'se_curl',
+    'net_zstd',
+    'org_tukaani_xz',
+    'com_google_brotli',
+    'org_sourceware_bzip2',
+    'org_nghttp2',
+    'com_github_nlohmann_json',
+    'org_aomedia_avif',
+    'com_github_cares_cares',
+    'libwebp',
+    'nasm',
+    'com_google_snappy',
+    'tinyxml2',
+    'org_blosc_cblosc',
+    'blake3'
+]
+
+# Generate list of includ paths and link path for syslibs to be passed to Bazel as --copt and --linkopt
```

Comment:
While we're waiting for a solution to the test problem, can you fix the typo here? "includ" -> include